lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -19,7 +19,10 @@ def show
   def update
     resource.validate_fields!
     if resource.errors.empty? && resource.update(user_params)
-      resource.kill_user_sessions(user_session) if resource.just_changed_password?
+      if resource.just_changed_password?
+        AuditLogService.call("User changed password: #{resource.id}/#{resource.username}")
```

Comment:
Don't we audit the whatever object stores the password already?

Reply:
I think it makes sense to start auditing that field. Just redact the value.

https://github.com/collectiveidea/audited/blob/9aad4d46e52625d1608917df22d0f31fbd873db4/lib/audited/auditor.rb#L40-L48